### DISCUSSION TITLE
Adding XPath to SpeakableSpecification

### DISCUSSION THREAD

**Discussion by twilliamsca** [RehanSaeed/Schema.NET#431] [Q&A] [answered]
If I want to have a SpeakableSpecification value for ["/html/head/meta[@name='description']/@content"] as a XPath value, how would I add to my Schema?

**Comment by Turnerj** [comment, marked as answer]
So [`XPath`](https://schema.org/XPathType) is a pending property/type for Schema.org so you'll need to use `Schema.NET.Pending` - this library contains all current + pending classes and properties. You can see it here: https://www.fuget.org/packages/Schema.NET.Pending/11.0.1/lib/netstandard2.0/Schema.NET.Pending.dll/Schema.NET/SpeakableSpecification

It would look something like:
```csharp
new SpeakableSpecification
{
    Xpath = "/html/head/meta[@name='description']/@content"
}
```

**Comment by content** [comment]
Please try not to ping me next time :)
